src/java.base/share/classes/java/lang/System.java

```diff
@@ -160,6 +160,7 @@ private System() {
     public static final PrintStream err = null;
 
     // indicates if a security manager is possible
+    // @implNote The HotSpot JVM hardcodes the value of NEVER.
```

Comment:
You don't need this if the VM reads the value of NEVER.

Reply:
ok.

Reply:
This is also good to me.